### ISSUE TITLE
Errors out at generating html report if not run in the project root

### ISSUE THREAD

**Issue by lalomartins** [klassijs/klassi-js#5]
My directory structure is exactly like the default, except everything is under one parent `e2e-tests`. I can run tests successfully if I do `node ./node_modules/webdriverio-cucumber-js/index.js -s ./e2e-tests/step-definitions -o ./e2e-tests/shared-objects -p ./e2e-tests/page-objects -r ./e2e-tests/reports` (and move `features` up to the root).

What I would like to do is to have `features` inside `e2e-tests` and run the whole thing with `node ../node_modules/webdriverio-cucumber-js/index.js`. That fails like so:

```
D:\Lalo\Projects\nowtilus\nemo-bridge\frontend\node_modules\webdriverio-cucumber-js\node_modules\cucumber\lib\cucumber\runtime\event_broadcaster.js:30
            throw error;
            ^

Error: ..\node_modules\webdriverio-cucumber-js\runtime\world.js:171 ENOENT: no such file or directory, open 'D:\Lalo\Projects\nowtilus\nemo-bridge\frontend\e2e-tests\package.json'
    at Error (native)
    at Object.fs.openSync (fs.js:640:18)
    at Object.fs.readFileSync (fs.js:508:33)
    at Object.readFileSync (D:\Lalo\Projects\nowtilus\nemo-bridge\frontend\node_modules\jsonfile\index.js:66:20)
    at generateReport (D:\Lalo\Projects\nowtilus\nemo-bridge\frontend\node_modules\webdriverio-cucumber-js\node_modules\cucumber-html-reporter\lib\reporter.js:14:32)
    at Object.generate (D:\Lalo\Projects\nowtilus\nemo-bridge\frontend\node_modules\webdriverio-cucumber-js\node_modules\cucumber-html-reporter\lib\reporter.js:235:9)
    at Object.generateReport [as generate] (D:\Lalo\Projects\nowtilus\nemo-bridge\frontend\node_modules\webdriverio-cucumber-js\node_modules\cucumber-html-reporter\index.js:30:21)
    at D:\Lalo\Projects\nowtilus\nemo-bridge\frontend\node_modules\webdriverio-cucumber-js\runtime\world.js:183:22
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

The root cause being that cucumber-html-reporter looks for my `package.json` in the cwd.

If I force-update cucumber-html-reporter to the latest version, running it the way I want just works. I didn't notice any other side-effects.

tl;dr: update cucumber-html-reporter dependency version. They're on 2.03 and what you have is 0.x. 😉

**Comment by lalomartins** [comment]
IRL that's not actually a huge deal for me, because as the paths indicate, these tests are supposed to be moved to a separate package/repository soon. But it's an annoying problem for now and way too easy to fix, so…

**Comment by larryg01** [comment]
I have been working on updating all the dependencies for the project, its nearing completion.
Thanks @lalomartins for the report
